Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2713,7 +2754,7 @@ script:
       description: A list of detection IDs connected to the incident.
       type: String
   - arguments: []
-    description: Returns the list of fields to map in outgoing mirroring. This command is only used for debugging purposes.
+    description: Returns the list of fields to map in outgoing mirroring. This command is only used for debugging purposes. Note that this command is supported in Cortex XSOAR only.
```

Comment:
can we hide these commands in XSIAM?

Reply:
it can be done by setting `hidden:marketplacev2: true`. However, this is not supported in the content item schema in the SDK so a validation error is raised. We can discuss offline if you would like more details.